skbeam/core/utils.py

```diff
@@ -1215,6 +1217,51 @@ def geometric_series(common_ratio, number_of_images, first_term=1):
 
     geometric_series = [first_term]
 
-    while geometric_series[-1]*common_ratio < number_of_images:
-        geometric_series.append(geometric_series[-1]*common_ratio)
+    while geometric_series[-1] * common_ratio < number_of_images:
+        geometric_series.append(geometric_series[-1] * common_ratio)
     return geometric_series
+
+
+def bin_grid(image, r_array, pixel_sizes, statistic='mean', mask=None,
```

Comment:
Be my guest! I'd do it myself but am very bandwidth limited at the moment. You might note that either the bins or the pixel size must be given. (This really should be refactored though, in light of the new binned statistic classes)